Aztec/Classes/Formatters/Implementations/ShadowBoldFormatter.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+/// ShadowBoldFormatter uses 2 different mechanisms to apply bold effect:
+/// 1. Headings: Applies shadow effect because headings are already bold by default
+/// 2. Other texts: Applies bold font trait. Uses BoldFormatter inside.
+class ShadowBoldFormatter: AttributeFormatter {
+    
+    private enum Shadow {
+        enum DefaultOffset {
+            static let width: CGFloat = 0.75
+            static let height: CGFloat = 0.0
+        }
+        
+        static let blurRadiusNoBlur: CGFloat = 0.0
+        
+        // Creates a no blur NSShadow instance with given offset values
+        static func shadow(width: CGFloat = DefaultOffset.width,
+                           height: CGFloat = DefaultOffset.height,
+                           color: UIColor = UIColor.black) -> NSShadow {
```

Comment:
@diegoreymendez 👋 The color choice is sth I want to ask, I couldn't find where to look for to see the what determines the color of headings? So I am thinking choosing black can be problematic in the future, shadow should actually be same as the font color, but not sure how to do it ? any help is appreciated

Reply:
I'd stick to the font color... and I'd probably avoid a default (in the method definition) if we can.

Considering both calls to this method have access to the string attributes, I'd pick up the color from there (and default there to black if the color is missing).